src/translations/en.json

```
@@ -2402,6 +2403,253 @@
           "gen_image_header": "Image generation tasks",
           "gen_image_description": "Generate images."
         },
+        "apps": {
+          "caption": "Apps",
+          "description": "Install and manage add-ons",
```

This probably needs to become "apps", too?

```suggestion
          "description": "Install and manage apps",
```